### PR TITLE
FEATURE: MSF-20118 Support PostgreSQL as input for LCM provisioning bricks

### PR DESCRIPTION
**Pull request by tqtu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/609/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/611/ : SUCCESS in 7s

File: lib/gooddata/cloud_resources/postgresql/postgresql_client.rb [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      end
+
+      def connect
+        GoodData.logger.info "Setting up connection to Postgresql #{@url}"
+
+        prop = java.util.Properties.new
+        prop.setProperty('user', @authentication['basic']['userName'])
+        prop.setProperty('password', @authentication['basic']['password'])
+        prop.setProperty('schema', @schema)
+        prop.setProperty('db', @database)
```
Comment (phong-nguyen-duy) [inline lib/gooddata/cloud_resources/postgresql/postgresql_client.rb:86]: no need to set database in properties, the url includes database already.

File: lib/gooddata/cloud_resources/postgresql/postgresql_client.rb [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        @connection = nil
+      end
+
+      def connect
+        GoodData.logger.info "Setting up connection to Postgresql #{@url}"
+
+        prop = java.util.Properties.new
+        prop.setProperty('user', @authentication['basic']['userName'])
+        prop.setProperty('password', @authentication['basic']['password'])
+        prop.setProperty('schema', @schema)
```
Comment (phong-nguyen-duy) [inline lib/gooddata/cloud_resources/postgresql/postgresql_client.rb:85]: Does it work? Do we need to execute "set search path..." to set schema?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/613/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/622/ : SUCCESS in 6s

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ruby-gate-PR1743

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-sonarqube-scan-zuul-el7/32/ : SUCCESS in 2m 46s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/513/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul-el7/71/ : FAILURE in 3m 08s (non-voting)

**Pull request merged**